### PR TITLE
Change default

### PR DESCRIPTION
**Pull request by jburel**
Make ice 3.6 the default downloadable versions
and remove the 3.5 versions from the download page

See https://trello.com/c/XvYIaZke/259-release-downloads

cc @sbesson @joshmoore

### CODE REVIEW

**Comment by manics** [comment]
Does this mean all our docs (omero-install, ansible roles) should also default to ice 3.6?

File: omero_downloads.html [hunk 1]
```diff
@@ -230,16 +230,6 @@
                                 <th width="122">Checksum</th>
                             </tr>
                             <tr>
-                                <td><a href="@SERVER35@"><img
-                                        src="images/server_ice_3_5.png"
```
Comment (hflynn) [inline omero_downloads.html:234]: These image files can be removed from the repo too

Comment (jburel) [reply omero_downloads.html:234]: @hflynn: done.
Also removed 3.4 ones

Comment (hflynn) [reply omero_downloads.html:234]: 👍 Thanks

**Comment by jburel** [comment]
ok to merge this PR?
Need to open another one for the download of zip

**Comment by hflynn** [comment]
No objections from me

**Review by hflynn — APPROVED** [review]

**Review by sbesson — APPROVED** [review]

**Pull request merged**